builders/build/eth-api/dev-env/brownie.md

```diff
@@ -0,0 +1,335 @@
+
+You will need to install Brownie and create a Brownie project if you don't already have one. You can choose to either create an empty project or use a [Brownie mix](https://eth-brownie.readthedocs.io/en/stable/init.html?highlight=brownie%20mix#creating-a-project-from-a-template){target=_blank}, which is essentially a template to build your project on. For this example, you can create an empty project. You can get started by completing the following steps:
+
+1. Create a directory for your project
+    ```
+    mkdir brownie && cd brownie
+    ```
+2. If you don't already have `pipx` installed, go ahead and install it
+    ```
+    python3 -m pip install --user pipx
+    python3 -m pipx ensurepath
+    ```
+
+2. [Install Brownie using `pipx`](https://eth-brownie.readthedocs.io/en/stable/install.html){target=_blank}. If you don't have `pipx` installed, you ca
+    ```
+    pipx install eth-brownie
+    ```
+    !!! note
+        [`pipx`](https://github.com/pypa/pipx){target=_blank} is used to run executables installed locally in your project. Brownie will be installed into a virtual environment and be available directly from the command line.
+3. Create an project
+    ```
+    brownie init
+    ```
+
+![Create Brownie project](/images/builders/build/eth-api/dev-env/brownie/brownie-1.png)
+
+Your Brownie project should contain the following empty directories:
+
+- **build** - for project data such as contract artifacts from compilation
+- **contracts** - to store the smart contract files
+- **interfaces** - for smart contract interfaces that are required for your project
+- **reports** - for JSON report files for use in the [Brownie GUI](https://eth-brownie.readthedocs.io/en/stable/gui.html){target=_blank}
+- **scripts** - where Python scripts used for deploying contracts or other automated tasks will live
+- **tests** - to store Python scripts for testing your project. Brownie uses the `pytest` framework for unit testing
+
+Another important file to note that is not included in an empty project is the `brownie-config.yaml` configuration file. The configuration file is optional and comes in handy when customizing specific settings such as a default network, compiler version and settings, and more. 
+
+## Network Configuration  {: #network-configuration } 
+
+To deploy to a Moonbeam network, you'll need to add and configure the network.
+
+Network configurations in Brownie are added from the command line. Brownie can be used with both development and live environments. Under the hood, Brownie uses Ganache for development environments. However, since a Moonbeam development node acts as your own personal development environment, Ganache isn't needed. Therefore, you can configure a development node the same way you would for a "live" network such as Moonriver or Moonbase Alpha.
+
+To add Moonriver, Moonbase Alpha, or a Moonbeam development node, you can run the following command:
+
+=== "Moonriver"
```

Comment:
Because they've already added Moonbeam as a network out of the box, so no need to go through the processing of adding one like you currently would have to with the other networks. I created a PR to add in these other networks, but it hasn't been merged yet. I'll try to see if I can push that along, but when I do get notification that the other networks have been added, I will be able to update the docs here